src/firebase-namespace.ts

```diff
@@ -48,6 +48,7 @@ export interface FirebaseServiceNamespace <T> {
  * Internals of a FirebaseNamespace instance.
  */
 export class FirebaseNamespaceInternals {
+  public CONFIG_FILE_VAR: string = 'FIREBASE_CONFIG';
```

Comment:
Please make this a constant. Also lets rename to something like `FIREBASE_CONFIG_ENV_VAR`.